### PR TITLE
Storage docs follow-up

### PR DESCRIPTION
**Pull request by xermicus**
Explain how to change the root key of contracts storage layout. @xgreenx can you think of situation where this is advisable, or why do we allow setting the key manually?

### CODE REVIEW

File: docs/datastructures/custom.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ pub struct ContractStorage {
 ```
 
 Even better: there is a macro
-[`#[ink::storage_item`](https://docs.rs/ink_macro/4.0.0-beta.1/ink_macro/attr.storage_item.html),
+[`#[ink::storage_item]`](https://docs.rs/ink_macro/4.0.0-beta.1/ink_macro/attr.storage_item.html),
```
Comment (SkymanOne) [inline docs/datastructures/custom.md:45]:
```suggestion
[`#[ink::storage_item]`](https://docs.rs/ink_macro/4.0.0-rc/ink_macro/attr.storage_item.html),
```

**Review by SkymanOne — CHANGES_REQUESTED** [review]
Awesome job again. Just need to update the link

**Comment by cmichi** [comment]
Can you update the files in https://github.com/paritytech/ink-docs/tree/master/i18n/es/docusaurus-plugin-content-docs/current/datastructures with the same changes?

For the English text, just wrap it in `<div class="translateTodo"></div>` as in https://github.com/paritytech/ink-docs/commit/211f0b87977c29b647ff3cc26d6dec37f2e76aa9?

**Review by cmichi — APPROVED** [review]

**Comment by xermicus** [comment]
@cmichi looks like the whole file needs to be translated (it has a TODO translate on top), so I omitted wrapping it in the `translateTodo` div.

**Review by cmichi — APPROVED** [review]

**Pull request merged**